common/test/keyboards/invalid/source/CERR_4040_balochi_phonetic_WithNoErrorThrownButErrorSpecified.kmn

```diff
@@ -0,0 +1,132 @@
+﻿c ***** This test will produce 1 Error: ****************************
+c ***** It will be marked as FAIL when run in kmcompxtest **********
+c *****    as ErrorNr (0x00004040) is specified but not thrown *****
```

Comment:
How is this different from `CERR_4040_balochi_phonetic_SpecifiedErrorNotThrown.kmn`?